### ISSUE TITLE
[CS 1.6/CZ] Repair bugged spectate animations.

### ISSUE THREAD

**Issue by JUMBO95** [ValveSoftware/halflife#2798]
Please repair bugged spectate animations. For example Desert eagle reloading when is changed to another player and back to player with desert eagle. Or reloading shotguns where spectating player dont see arm. There is also a bug with Shield where a player can shoot from a glock but spectating player see something indescribable. There is a bug when you play CZ with bots after pressing the windows button your game is minimized to the desktop but after every kill is the mouse cursor is reset to the upper left corner so its hard for example check something on internet.

**Comment by kisak-valve** [comment]
Hello @JUMBO95, in general, multi-issue issue reports are bad for any issue tracker, so only the Desert Eagle reload animation issue will be tracked here.

The mouse position issue sounds like #2635.

**Comment by JUMBO95** [comment]
So can i make new threat with reload animation issue?

**Comment by CS-PRO1** [comment]
> So can i make new threat with reload animation issue?

No, just remove the mouse issue from the description and it'll be enough.
